tools/topology/topology2/cavs-rt5682-rt1019.conf

```diff
@@ -0,0 +1,283 @@
+<include/common/vendor-token.conf>
+<include/common/tokens.conf>
+<include/common/manifest.conf>
+<include/common/pcm.conf>
+<include/common/pcm_caps.conf>
+<include/common/fe_dai.conf>
+<include/common/route.conf>
+<include/components/host.conf>
+<include/components/dai.conf>
+<include/components/pipeline.conf>
+<include/components/copier.conf>
+<include/dais/ssp.conf>
+<include/dais/dmic.conf>
+<include/dais/pdm_config.conf>
+<include/common/data.conf>
+<include/dais/hw_config.conf>
+<include/pipelines/cavs/passthrough-capture.conf>
+<include/pipelines/volume-playback.conf>
+<include/pipelines/volume-capture.conf>
+
+#
+# List of all DAIs
+#
+# PCM0 ----> volume -----> SSP1  (Speaker - ALC1019)
+# PCM1 <---> volume <----> SSP0  (Headset - ALC5682)
+# PCM3 <---- volume <---- DMIC01 (dmic 48k capture)
+Object.Dai {
+	SSP."0" {
+		id 		0
+		direction	"duplex"
+		name		SSP0-Codec
+		default_hw_conf_id	0
+		sample_bits		16
+
+		Object.Base.hw_config."SSP0" {
+			id	0
+			mclk_freq	24000000
+			bclk_freq	4800000
+			tdm_slot_width	32
```

Comment:
that seems wrong. this would give you a 150kHz frame rate.

Reply:
updated the changes - referring to
https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof-jsl-rt5682.m4#L123-L135
https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof-jsl-rt5682.m4#L176-L181